spec/game_spec.rb

```diff
@@ -0,0 +1,80 @@
+# game_spec.rb
+
+require './lib/player'
+require './lib/game'
+
+RSpec.describe Game do
+  let(:game) { Game.new }
+  let(:player) { Player.new('Manuel', 'X') }
+
+  describe '#move' do
+    context 'when the method is called' do
+      it 'sets a bunch of variables according to comparisons and returns nil always' do
+        expect(game.move(player, 5)).to be(nil)
+      end
+    end
+  end
+
+  describe '#free?' do
+    context 'when the method is called' do
+      it 'returns true if the number given as argument is available in the board' do
+        expect(game.free?(1)).to be(true)
+      end
+
+      it "doesn't return true if the number given as argument isn't available in the board" do
+        game.move(player, 5)
+        expect(game.free?(5)).not_to be(true)
+      end
+    end
+  end
+
+  describe '#board' do
+    context 'when the method is called' do
+      it 'returns a string with the current board status' do
+        expect(game.board).to be_an_instance_of(String)
```

Comment:
You should add test to check that the string returned correctly represents the state of the board.